### PR TITLE
Service style

### PR DESCRIPTION
**Pull request by allenyhu**
Mostly formatting changes from clang-format. Some additions for the actual serviceLayer.h / .c files for the stream method

### CODE REVIEW

**Review by pinghaoluo — APPROVED** [review]
Looks fine to me

**Comment by pinghaoluo** [comment]
My serviceServer.cc is serviceLayer working with GRPC and GRPC keyValue Store and my serviceLayer.cc is serviceLayer instance working with keyValue Store Object. You may want to implement serviceServer.cc first.

File: serviceLayer.h [hunk 1]
```diff
@@ -1,71 +1,69 @@
...
+  int getID(KeyValueMap &store);
+
+  int chirp(std::string user1, std::string chirp, std::string parent,
+            KeyValueMap &store);
+
+  std::vector<std::string> read(std::string chirp_id, KeyValueMap &stor);
 
+  std::vector<std::string> monitor(std::string user1, KeyValueMap &store);
 
+  std::vector<std::string> stream(std::string user, std::string hashtag);
```
Comment (barath) [inline serviceLayer.h]: You should at least add a comment for the new method you've added.  Also, const references.

**Review by barath — APPROVED** [review]

File: serviceLayer.h (hunk 1 above)
Comment (allenyhu) [reply serviceLayer.h]: done

**Pull request merged**